cs2log_test.go

```diff
@@ -1,22 +1,25 @@
-package cs2log
+package cs2log_test
 
 import (
 	"fmt"
 	"regexp"
+	"strconv"
 	"strings"
 	"testing"
 	"time"
+
+	cs2log "github.com/janstuemmel/cs2-log"
 )
 
-func ExampleParse() {
+func exampleParse() {
```

Comment:
does this show up as an example in godoc?

Reply:
I do not think so, maybe we could use [this way](https://github.com/googleapis/google-cloud-go/blob/main/compute/apiv1/doc.go) instead ?
https://pkg.go.dev/cloud.google.com/go/compute@v1.23.3/apiv1

Reply:
that does not work as test example, the example gets tested while running `go test -v ./...` and the assertion is the comment below the log

Reply:
666ed65edaac307a6973d1953b0f9a1dc719ea5f